tests/e2e/africas-talking.js

```diff
@@ -160,39 +160,37 @@ describe('africas talking api', () => {
       });
     };
 
-    beforeEach(done => {
-      submitSms({ from: '+64271234567', text: 'hello', id: 'a' })
-        .then(done)
-        .catch(done.fail);
+    beforeEach(() => {
+      return submitSms({ from: '+64271234567', text: 'hello', id: 'a' });
```

Comment:
We can make this a nice one line by omitting the return.

`beforeEach(() => submitSms({ from: '+64271234567', text: 'hello', id: 'a' }));`